### ISSUE TITLE
PHP errors and SFTP details not showing in dashboard

### ISSUE THREAD

**Issue by backamblock** [littlebizzy/slickstack#137]
Hi there, i installed slickstack several times now but the errors are still there.
It is a fresh VPS from Hetzner with Ubuntu 20.4 and both 1c/2gb and 2c/2gb tried
i ran apt update && apt upgrade before so it should all be up to date.

<img width="544" alt="image" src="https://user-images.githubusercontent.com/20801141/150049757-7736bc90-7736-4cc2-b6c6-f6f976dd8133.png">

is this expected when not running on your littlebizzy managed servers or should it still work?

**Comment by backamblock** [comment]
also the php error log has some entries

`[19-Jan-2022 02:08:31 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:08:31 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:08:30 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:08:30 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:08:02 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:08:02 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:08:02 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:08:02 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:08:02 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:07:19 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:07:18 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:07:18 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:07:18 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:07:18 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:07:18 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:07:18 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:18 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:06:18 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:18 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:18 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:18 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:17 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:06:17 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:17 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:16 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:16 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:01 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:06:01 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:01 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:01 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:06:01 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:05:57 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:05:57 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:05:57 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:05:57 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:05:57 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:04:01 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:04:00 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:04:00 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:04:00 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:04:00 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:59 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:58 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:58 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:58 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:58 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:58 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:57 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:57 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:57 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:57 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:57 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:56 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:56 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:56 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:56 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:55 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:55 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:55 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:55 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:55 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:54 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:54 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:54 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:54 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:54 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:46 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:45 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:45 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:45 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:45 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:41 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:41 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:41 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:41 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Notice:  Undefined variable: modified in /var/www/html/wp-content/mu-plugins/force-https/helpers/updater.php on line 136
[19-Jan-2022 02:02:40 UTC] PHP Notice:  Undefined variable: modified in /var/www/html/wp-content/mu-plugins/clear-caches/helpers/updater.php on line 136
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Notice:  Undefined variable: modified in /var/www/html/wp-content/mu-plugins/force-https/helpers/updater.php on line 136
[19-Jan-2022 02:02:40 UTC] PHP Notice:  Undefined variable: modified in /var/www/html/wp-content/mu-plugins/clear-caches/helpers/updater.php on line 136
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:40 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:38 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:38 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:38 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:38 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:34 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:34 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:33 UTC] PHP Notice:  Undefined variable: modified in /var/www/html/wp-content/mu-plugins/force-https/helpers/updater.php on line 136
[19-Jan-2022 02:02:33 UTC] PHP Notice:  Undefined variable: modified in /var/www/html/wp-content/mu-plugins/clear-caches/helpers/updater.php on line 136
[19-Jan-2022 02:02:33 UTC] PHP Notice:  Undefined variable: modified in /var/www/html/wp-content/mu-plugins/force-https/helpers/updater.php on line 136
[19-Jan-2022 02:02:33 UTC] PHP Notice:  Undefined variable: modified in /var/www/html/wp-content/mu-plugins/clear-caches/helpers/updater.php on line 136
[19-Jan-2022 02:02:32 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:32 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:32 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:32 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:32 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
sh: 1: /usr/sbin/sendmail: not found
[19-Jan-2022 02:02:32 UTC] PHP Warning:  Invalid argument supplied for foreach() in /var/www/html/wp-cron.php on line 117
[19-Jan-2022 02:02:31 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:31 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:31 UTC] PHP Notice:  Array to string conversion in /var/www/html/wp-content/mu-plugins/autoloader.php on line 163
[19-Jan-2022 02:02:31 UTC] PHP Notice:  Trying to access array offset on value of type null in /var/www/html/wp-content/mu-plugins/autoloader.php on line 144
[19-Jan-2022 02:02:31 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:31 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:30 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:02:30 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:01:38 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:01:38 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:01:33 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:01:33 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455
[19-Jan-2022 02:01:33 UTC] PHP Warning:  include_once(): Failed opening '/var/www/html/wp-content/functions.php' for inclusion (include_path='.:/usr/share/php') in /var/www/html/wp-config.php on line 455
9-Jan-2022 02:01:33 UTC] PHP Warning:  include_once(/var/www/html/wp-content/functions.php): failed to open stream: No such file or directory in /var/www/html/wp-config.php on line 455`

**Comment by jessuppi** [comment]
The SlickStack dashboard errors should be fixed now for SFTP tab and others. In the past week I did a lot of cleanup on the files involved like `ss-constants.php` and `ss-worker` and `xxx-common.php` etc... a lot of small commits and nothing really worth linking here, but just mentioned that it's fixed now.

https://example.com/wp-admin/admin.php?page=slickstack&tab=sftp

(etc)

The failed opening of `/wp-content/functions.php` is a separate issue, but also fixed now. That was leftover code in our boilerplate for `wp-config.php` that was calling custom functions... we deprecated that a few months ago.
